src/python_framework_v02/nhd_io.py

```diff
@@ -39,9 +39,9 @@ def read_mask(path, layer_string=None):
     return read_csv(path, header=None, layer_string=layer_string)
 
 
-def read_custom_input_json(custom_input_file):
-    with open(custom_input_file) as json_file:
-        data = json.load(json_file)
+def read_custom_input_yaml(custom_input_file):
+    with open(custom_input_file) as yaml_file:
+        data = yaml.load(yaml_file)
```

Comment:
Instead of removing the `read_custom_input_json` function, let's make an option for either -fjson or -fyaml and read either one....